### PR TITLE
Adds Kharadron Overlords 2023 battletome

### PR DESCRIPTION
**Pull request by PixelTom**
Adds Kharadron Overlords 2023 Battletome.

The failing tests is a bit over my head, lots of things have changed between versions.

Would love some help on how to update tests to work correctly.

### CODE REVIEW

**Comment by daviseford** [comment]
@PixelTom let me know when you want a second set of eyes!

**Comment by PixelTom** [comment]
Thanks @daviseford, unfortunately my wife and son have been ill so they have taken up my time. Will try to get back on this soon.

**Comment by PixelTom** [comment]
That should be everything @daviseford 

Have some failing tests. And I know why. But unsure on the best way to update them.

File: src/factions/kharadron_overlords/battle_traits.ts [hunk 1]
```diff
@@ -1,50 +1,57 @@
...
 const BattleTraits = {
   [KHARADRON_OVERLORDS]: {
     effects: [
-      {
-        name: `Aether-gold`,
-        desc: `Each KHARADRON OVERLORDS HERO, SKYVESSEL and KHARADRON OVERLORDS unit with 10 or more models starts a battle with 1 share of aether-gold. Once per phase, you can say that 1 unit from your army that has any shares of aether-gold will spend 1 of them. If you do so, subtract 1 from that unit's Bravery characteristic for the rest of the battle, but you can pick a triumph it is eligible to use and immediately apply its effect to that unit. Ignore any restrictions on a triumph that say it can only be used once per battle if you pay to use it with a share of aether-gold.`,
-        when: [DURING_GAME],
-      },
+      // Should I put Embark and Disembark in here?
+      // Yes, if it's an army-wide rule that's always in effect
```
[the remark this answers is not in the document]
Comment (PixelTom) [reply src/factions/kharadron_overlords/battle_traits.ts:15]: Ah, looking at that again @daviseford it's always in effect ONLY if there are SKYVESSELS in the army list. 

Now... it's quite rare to have a 1k+ list without any boats, but it does happen.

Should it be added there still anyway? Or should we cater for the edge cases of boatless lists?

**Pull request merged**